kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/KFuzzConfig.kt

```diff
@@ -27,6 +32,8 @@ interface KFuzzConfig {
     val instrument: List<String>
```

Comment:
Change `keepGoing` type to `Long`. Since we are casting it anyway

Reply:
Fixed